### ISSUE TITLE
setState() called after dispose():

### ISSUE THREAD

**Issue by Emon526** [subhamayd2/day_night_time_picker#96]
After updating to 1.2.0 .when ever any picker changes it shows the error

════════ Exception caught by scheduler library ═════════════════════════════════
setState() called after dispose(): TimeModelBindingState#f28dc(lifecycle state: defunct, not mounted)

**Comment by subhamayd2** [comment]
I was not able to recreate this. Can you provide more info on this, like your code setup, etc?

**Comment by arsyad-revo** [comment]
i just got this issues, seems appear when try to put ios style in dialog example app
```
TextButton(
                  style: TextButton.styleFrom(
                    backgroundColor: Theme.of(context).colorScheme.secondary,
                  ),
                  onPressed: () {
                    Navigator.of(context).push(
                      showPicker(
                        elevation: 1,
                        value: _time,
                        onChange: onTimeChanged,
                        minuteInterval: TimePickerInterval.FIVE,
                        iosStylePicker: iosStyle,
                        minHour: 9,
                        maxHour: 21,
                        is24HrFormat: false,
                      ),
                    );
                  },
                  child: const Text(
                    "Open time picker",
                    style: TextStyle(color: Colors.white),
                  ),
                ),
```

**Comment by hendrait** [comment]
this issue happens to me too. I put the the picker inside show dialog and using iostyle. When i change to default style its work fine

**Comment by WillianSalceda** [comment]
Same problem. Reproduces when using iOS style (1.2.0+1).

**Comment by subhamayd2** [comment]
This should be fixed in the latest release `v1.2.0+2`
